### PR TITLE
S6 search bar

### PR DESCRIPTION
**Pull request by Benson-J**
Added a search bar & reset button at top of page

To be reviewed:

index.php
 - First form

index.scss
 - body 
 - main (code here was moved from #scroll, to fit search bar into main structure)
 - .nameSearch
 - .reset
 - .button (code here was moved from #save, to allow it to be used generically)

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<head>
+    <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Cantarell:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/index.css">
+    <title>Pokédex</title>
+</head>
+<body>
+    <main>
+        <form action="index.php" class="nameSearch">
```
Comment (hoptond) [inline index.php:29]: This should probably be lined up with the ul beneath it and every element in the page should be the same size, as well as styled. Compare with the form elements on the login page.

File: index.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            <input type="search" name="nameSearch" placeholder="Search by name">
```
Comment (hoptond) [inline index.php]: Minor point, but the get variable I used in the branch that does the legwork is just called "search". You might want to rename it to that to make the mergening easier

**Review by hoptond — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (Benson-J) [reply index.php:29]: Changed in e094b46.

File: index.php (hunk 2 above)
Comment (Benson-J) [reply index.php]: Changed in f7c4057.

**Review by hoptond — APPROVED** [review]

**Pull request merged**